src/jsConnect/Extensions.cs

```diff
@@ -26,7 +26,11 @@ public static string ToJsonp(this object o, string callback)
 			{
 				throw new ArgumentNullException(nameof(o));
 			}
-			return $"{callback}({JsonConvert.SerializeObject(o)})";
+			var settings = new JsonSerializerSettings
+			{
+				NullValueHandling = NullValueHandling.Ignore
```

Comment:
just one question @rudism 

I'm wondering whether it's better to have the JSON settings hardcoded here or whether to leave it up to the user as described in https://github.com/petrsvihlik/jsConnectAspNetCoreMvc#getting-started

I think your approach might be better in that you don't have to change the JSON configuration for the whole app but just for JsonP serialization here.

What are your thoughts?

Reply:
if you agree, I'd suggest removing that part from the readme

Reply:
@petrsvihlik The `JsonSerializerOptions` mentioned in the readme don't seem to affect the `JsonConvert` functions anyway (I had `IgnoreNullValues` set to `true` but they were still being excluded in the JsonP) so it probably makes sense to remove that anyway.

I also think localizing the settings to where it's needed is probably better, since the application may have other JSON services elsewhere that could be affected by a global configuration change.

Reply:
ok, agreed